src/components/Checkout/DonationNudgeLightbox.vue

```diff
@@ -0,0 +1,175 @@
+					</div>
+				</div>
+				<kv-donation-nudge-boxes
+					id="nudge-donation-top-boxes-wrapper"
+					:percentage-rows= "percentageRows"
+					:has-custom-donation="hasCustomDonation"
+					:loan-reservation-total="loanReservationTotal"
+					:set-donation-and-close="setDonationAndClose"
+				/>
+				<div><a id="no-donation-link" @click="setDonationAndClose(0)">No donation to Kiva</a></div>
+			</div>
+			<div id="nudge-donation-bottom" class="show-for-large">
+				<div class="row">
+					<div class="large-8 xlarge-6 large-offset-2 xlarge-offset-3 columns">
+						<kv-charity-navigator
+							title="Kiva has been awarded Charity Navigator's highest rating for operational efficiency."
+							:wide-icon="true"
+							subtitle="Your donation is eligible for a tax deduction if you live in the U.S."
+						/>
+					</div>
+				</div>
+			</div>
+		</div>
+	</kv-lightbox>
+</template>
+
+<script>
+import KvButton from '@/components/Kv/KvButton';
+import KvLightbox from '@/components/Kv/KvLightbox';
+import KvDonationNudgeBoxes from '@/components/Checkout/KvDonationNudgeBoxes';
+import KvCharityNavigator from '@/components/Kv/KvCharityNavigator';
+
+export default {
+	components: {
+		KvButton,
+		KvLightbox,
+		KvCharityNavigator,
+		KvDonationNudgeBoxes,
+	},
+	props: {
+		nudgeLightboxVisible: {
+			type: Boolean,
+			required: true,
+		},
+		closeNudgeLightbox: {
+			type: Function,
+			required: true,
+		},
+		updateDonationTo: {
+			type: Function,
+			required: true,
+		},
+		loanCount: {
+			type: Number,
+			default: 0
+		},
+		loanReservationTotal: {
+			type: Number,
+			default: 0,
+		},
+		hasCustomDonation: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	computed: {
+		percentageRows() {
+			const basePercentageRows = [
+				{
+					percentage: 15,
+					appeal: `Cover the cost to facilitate ${this.loanCount > 1 ? 'these loans' : 'this loan'}`,
+					appealIsHorizontallyPadded: false,
+				},
+				{
+					percentage: 20,
+					appeal: 'Reach more people around the world!',
+					appealIsHorizontallyPadded: false,
+				},
+			];
+			const lowPercentage = [{
+				percentage: 10,
+				appeal: 'Cover some of Kiva\'s costs',
+				appealIsHorizontallyPadded: true,
+			}];
+			return this.hasCustomDonation ? basePercentageRows : lowPercentage.concat(basePercentageRows);
+		},
+	},
+	methods: {
+		setDonationAndClose(amount) {
+			this.updateDonationTo(amount);
+			this.$kvTrackEvent('basket', 'Update Loan Amount', 'Update Success', amount * 100);
+			this.closeNudgeLightbox();
+		}
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+#nudge-donation-container {
+	.nudge-lightbox-row-padding {
+		padding-left: 3rem;
+		padding-right: 3rem;
+	}
+
+	text-align: center;
+
+	#nudge-donation-top {
+		padding-bottom: 3rem;
+		background: #F8F8F8;
```

Comment:
@BoulderBrains @ksekhri Please keep an eye out for these colors being requested in other new designs... They really should be using colors in the palette from our styleguide rather than breaking consistency to infinitely expand the palette.

If they start showing up in other designs we need to start replacing the values for existing variables so these colors are applied site wide.